templates/terraform/resource.erb

```diff
@@ -143,7 +169,11 @@ func resource<%= resource_name -%>Read(d *schema.ResourceData, meta interface{})
 		return handleNotFoundError(err, d, fmt.Sprintf("<%= resource_name -%> %q", d.Id()))
 	}
 <% properties.each do |prop| -%>
+  <% if object.custom_code.decoder.key? prop.name -%>
+	if err := d.Set("<%= Google::StringUtils.underscore(prop.name) -%>", flatten<%= resource_name -%><%= titlelize_property(prop) -%>(customDecoder<%= resource_name -%><%= titlelize_property(prop) -%>(res["<%= prop.api_name -%>"], d, config))); err != nil {
```

Comment:
nit: this is a pretty long line, anything we can do to make it a bit easier to read?

Reply:
This seems better to me.